kubejs/data/enigmatica/trofers/tame_all_animals.json

```diff
@@ -0,0 +1,38 @@
+{
+    "name": {
+        "text": "Zoology Trophy",
+        "color": "#f91d80"
+    },
+    "tooltip": [
+        {"text": "Tame all animals! And here is your special cat for that."}
```

Comment:
`And here is your special cat for that.` what?

Reply:
The trophy is literally a cat on purple carpet 😅
![obraz](https://user-images.githubusercontent.com/60540476/211215843-04a05e32-47fa-49e2-a884-41bbc2f7e53a.png)

`And here is your special cat for that.` -> `And here is your own special cat as a reward.` Maybe ^^"? Or I should just remove that bit.

Reply:
Just remove that bit imo

Reply:
Alrighty.